modules/web-console-query-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * web_console/web-console-query-cli.adoc
+
+[id="web-console-query-cli_{context}"]
+= Query image vulnerabilities from the CLI
+Using the `oc` command, you can display information about
+vulnerabilities detected by the Container Security Operator.
+
+.Prerequisites
+* Be running the Container Security Operator on your
```

Comment:
Suggest rewording "Be running". How about something like "The Container Security Operator is installed in your cluster" ?

Reply:
That's fine.

Reply:
Actually, no. The point is that this procedure is only useful if you are running containers that come from a Red Hat Quay or Quay.io registry. I reworded as follows: "Have containers that come from a Red Hat Quay or Quay.io registry running on your cluster"